Vagrantfile

```diff
@@ -54,6 +54,10 @@ Vagrant.configure("2") do |config|
       override.vm.box_url = "http://%s.release.core-os.net/amd64-usr/current/coreos_production_vagrant_vmware_fusion.json" % $update_channel
     end
   end
+  config.vm.provider :parallels do |vb, override|
```

Comment:
We need to generate parallels boxes and host them on the normal storage site (release.core-os.net) as a part of our normal build process. Maybe @marineam or @crawford can help with that.